input/dir/dir_input.go

```diff
@@ -0,0 +1,55 @@
+package dir
+
+import (
+	"crypto/sha512"
+	"hash"
+	"os"
+
+	"github.com/spacemonkeygo/errors"
+	"polydawn.net/repeatr/def"
+	"polydawn.net/repeatr/input"
+	"polydawn.net/repeatr/lib/fshash"
+)
+
+const Type = "dir"
+
+// interface assertion
+var _ input.Input = &Input{}
```

Comment:
I'm mainly thinking ahead to whenever we might refactor an interface and be unable to build until we adapt all implementations. *shrug*

Let's leave it for now, can always remove later.